### DISCUSSION TITLE
Only Allowing Search of Amazon and Bestbuy

### DISCUSSION THREAD

**Discussion by MJB1113** [jef/streetmerchant#1755] [Help]
## Expected Behavior

There should be 13 stores that are being searched

## Current Behavior

I have more than two stores selected however only those are showing up (in the picture below)

## Steps to Reproduce

![image](https://user-images.githubusercontent.com/45674525/103664390-3c1d8180-4f40-11eb-8ae6-131a19a2f866.png)

## Environment

**OS:**
**dotenv file:**

```dotenv

```

**Comment by wofnull** [comment]
how about you post the contents of the dotenv file? dotenv not dotenv-example!

**Comment by MJB1113** [comment]
> how about you post the contents of the dotenv file? dotenv not dotenv-example!

# ** All configuration variables are optional **
# Copy this file and name it as `dotenv` (without backticks)
# Read https://jef.codes/streetmerchant/getting-started/#customization for help on customizing this file
########################################################################################################

ASCII_BANNER=
ASCII_COLOR=
AUTO_ADD_TO_CART=true
BROWSER_TRUSTED=
COUNTRY=USA
DESKTOP_NOTIFICATIONS=true
DISCORD_NOTIFY_GROUP=
DISCORD_NOTIFY_GROUP_3060TI=true
DISCORD_NOTIFY_GROUP_3070=true
DISCORD_NOTIFY_GROUP_3080=true
DISCORD_NOTIFY_GROUP_3090=true
DISCORD_NOTIFY_GROUP_CORSAIR_SF=
DISCORD_NOTIFY_GROUP_RX6800=
DISCORD_NOTIFY_GROUP_RX6800XT=
DISCORD_NOTIFY_GROUP_RX6900XT=
DISCORD_NOTIFY_GROUP_RYZEN5600=
DISCORD_NOTIFY_GROUP_RYZEN5800=
DISCORD_NOTIFY_GROUP_RYZEN5900=
DISCORD_NOTIFY_GROUP_RYZEN5950=
DISCORD_NOTIFY_GROUP_SONYPS5C=
DISCORD_NOTIFY_GROUP_SONYPS5DE=
DISCORD_NOTIFY_GROUP_XBOXSX=
DISCORD_NOTIFY_GROUP_XBOXSS=
DISCORD_NOTIFY_GROUP_TEST=
DISCORD_WEB_HOOK=https://discord.com/api/webhooks/794820191489228821/4f5toLJBdHc6YfTPN65fMa1cPDLByG9ozqNSiqfg5B3GHcTgX-i9F9Y1iWsG6WmsiDVw
EMAIL_PASSWORD=
EMAIL_TO=
EMAIL_USERNAME=
HEADLESS=
IN_STOCK_WAIT_TIME= 600
LOG_LEVEL=
LOW_BANDWIDTH=
MAX_PRICE_SERIES_3060TI=650
MAX_PRICE_SERIES_3070=750
MAX_PRICE_SERIES_3080=850
MAX_PRICE_SERIES_3090=1800
MAX_PRICE_SERIES_CORSAIR_SF=
MAX_PRICE_SERIES_RX6800=
MAX_PRICE_SERIES_RX6800XT=
MAX_PRICE_SERIES_RX6900XT=
MAX_PRICE_SERIES_RYZEN5600=
MAX_PRICE_SERIES_RYZEN5800=
MAX_PRICE_SERIES_RYZEN5900=
MAX_PRICE_SERIES_RYZEN5950=
MAX_PRICE_SERIES_SONYPS5C=
MAX_PRICE_SERIES_SONYPS5DE=
MAX_PRICE_SERIES_XBOXSS=
MAX_PRICE_SERIES_XBOXSX=
MAX_PRICE_SERIES_TEST=
MICROCENTER_LOCATION=
MQTT_BROKER_ADDRESS=
MQTT_BROKER_PORT=
MQTT_CLIENT_ID=
MQTT_PASSWORD=
MQTT_QOS=
MQTT_TOPIC=
MQTT_USERNAME=
NVIDIA_ADD_TO_CART_ATTEMPTS=150
NVIDIA_SESSION_TTL=60
OPEN_BROWSER=true
PAGE_BACKOFF_MIN=
PAGE_BACKOFF_MAX=
PAGE_SLEEP_MIN=
PAGE_SLEEP_MAX=
PAGE_TIMEOUT=
PAGERDUTY_INTEGRATION_KEY=
PAGERDUTY_SEVERITY=
PHILIPS_HUE_API_KEY=
PHILIPS_HUE_CLOUD_ACCESS_TOKEN=
PHILIPS_HUE_CLOUD_CLIENT_ID=
PHILIPS_HUE_CLOUD_CLIENT_SECRET=
PHILIPS_HUE_CLOUD_REFRESH_TOKEN=
PHILIPS_HUE_LAN_BRIDGE_IP=
PHILIPS_HUE_LIGHT_COLOR=
PHILIPS_HUE_LIGHT_IDS=
PHILIPS_HUE_LIGHT_PATTERN=
PHONE_CARRIER=tmobile
PHONE_NUMBER=
PLAY_SOUND=
PROXY_ADDRESS=
PROXY_PROTOCOL=
PROXY_PORT=
PUSHBULLET=
PUSHOVER_EXPIRE=
PUSHOVER_RETRY=
PUSHOVER_TOKEN=
PUSHOVER_USER=
PUSHOVER_PRIORITY=
SCREENSHOT=true
SHOW_ONLY_BRANDS=
SHOW_ONLY_MODELS=
SHOW_ONLY_SERIES=
SLACK_CHANNEL=
SLACK_TOKEN=
SMTP_ADDRESS=
SMTP_PORT=
STORES=amazon,amd,adorama,antonline,bestbuy,asus,bandh,corsair,evga,gamestop,newegg,nvidia,officedepot
TELEGRAM_ACCESS_TOKEN=
TELEGRAM_CHAT_ID=
TWILIO_ACCOUNT_SID=
TWILIO_AUTH_TOKEN=
TWILIO_FROM_NUMBER=
TWILIO_TO_NUMBER=
TWITCH_ACCESS_TOKEN=
TWITCH_CHANNEL=
TWITCH_CLIENT_ID=
TWITCH_CLIENT_SECRET=
TWITCH_REFRESH_TOKEN=
TWITTER_ACCESS_TOKEN_KEY=
TWITTER_ACCESS_TOKEN_SECRET=
TWITTER_CONSUMER_KEY=
TWITTER_CONSUMER_SECRET=
TWITTER_TWEET_TAGS=
WEB_PORT=

**Comment by DigiBugCat** [comment]
I believe the issue here is you did not surround the store list in quotes. It should be something like the following: `STORES="amazon,amd,adorama,antonline,bestbuy,asus,bandh,corsair,evga,gamestop,newegg,nvidia,officedepot"`

**Comment by MJB1113** [comment]
> I believe the issue here is you did not surround the store list in quotes. It should be something like the following: `STORES="amazon,amd,adorama,antonline,bestbuy,asus,bandh,corsair,evga,gamestop,newegg,nvidia,officedepot"`

this didn't work :(

**Comment by DigiBugCat** [comment]
Interesting, can you repost your updated dotenv file?

**Comment by MJB1113** [comment]
> Interesting, can you repost your updated dotenv file?
** All configuration variables are optional **
Copy this file and name it as dotenv (without backticks)
Read https://jef.codes/streetmerchant/getting-started/#customization for help on customizing this file
########################################################################################################

ASCII_BANNER=
ASCII_COLOR=
AUTO_ADD_TO_CART=true
BROWSER_TRUSTED=
COUNTRY=USA
DESKTOP_NOTIFICATIONS=true
DISCORD_NOTIFY_GROUP=
DISCORD_NOTIFY_GROUP_3060TI=true
DISCORD_NOTIFY_GROUP_3070=true
DISCORD_NOTIFY_GROUP_3080=true
DISCORD_NOTIFY_GROUP_3090=true
DISCORD_NOTIFY_GROUP_CORSAIR_SF=
DISCORD_NOTIFY_GROUP_RX6800=
DISCORD_NOTIFY_GROUP_RX6800XT=
DISCORD_NOTIFY_GROUP_RX6900XT=
DISCORD_NOTIFY_GROUP_RYZEN5600=
DISCORD_NOTIFY_GROUP_RYZEN5800=
DISCORD_NOTIFY_GROUP_RYZEN5900=
DISCORD_NOTIFY_GROUP_RYZEN5950=
DISCORD_NOTIFY_GROUP_SONYPS5C=
DISCORD_NOTIFY_GROUP_SONYPS5DE=
DISCORD_NOTIFY_GROUP_XBOXSX=
DISCORD_NOTIFY_GROUP_XBOXSS=
DISCORD_NOTIFY_GROUP_TEST=
DISCORD_WEB_HOOK=https://discord.com/api/webhooks/794820191489228821/4f5toLJBdHc6YfTPN65fMa1cPDLByG9ozqNSiqfg5B3GHcTgX-i9F9Y1iWsG6WmsiDVw
EMAIL_PASSWORD=
EMAIL_TO=
EMAIL_USERNAME=
HEADLESS=
IN_STOCK_WAIT_TIME= 600
LOG_LEVEL=
LOW_BANDWIDTH=
MAX_PRICE_SERIES_3060TI=650
MAX_PRICE_SERIES_3070=750
MAX_PRICE_SERIES_3080=850
MAX_PRICE_SERIES_3090=1800
MAX_PRICE_SERIES_CORSAIR_SF=
MAX_PRICE_SERIES_RX6800=
MAX_PRICE_SERIES_RX6800XT=
MAX_PRICE_SERIES_RX6900XT=
MAX_PRICE_SERIES_RYZEN5600=
MAX_PRICE_SERIES_RYZEN5800=
MAX_PRICE_SERIES_RYZEN5900=
MAX_PRICE_SERIES_RYZEN5950=
MAX_PRICE_SERIES_SONYPS5C=
MAX_PRICE_SERIES_SONYPS5DE=
MAX_PRICE_SERIES_XBOXSS=
MAX_PRICE_SERIES_XBOXSX=
MAX_PRICE_SERIES_TEST=
MICROCENTER_LOCATION=
MQTT_BROKER_ADDRESS=
MQTT_BROKER_PORT=
MQTT_CLIENT_ID=
MQTT_PASSWORD=
MQTT_QOS=
MQTT_TOPIC=
MQTT_USERNAME=
NVIDIA_ADD_TO_CART_ATTEMPTS=150
NVIDIA_SESSION_TTL=60
OPEN_BROWSER=true
PAGE_BACKOFF_MIN=
PAGE_BACKOFF_MAX=
PAGE_SLEEP_MIN=
PAGE_SLEEP_MAX=
PAGE_TIMEOUT=
PAGERDUTY_INTEGRATION_KEY=
PAGERDUTY_SEVERITY=
PHILIPS_HUE_API_KEY=
PHILIPS_HUE_CLOUD_ACCESS_TOKEN=
PHILIPS_HUE_CLOUD_CLIENT_ID=
PHILIPS_HUE_CLOUD_CLIENT_SECRET=
PHILIPS_HUE_CLOUD_REFRESH_TOKEN=
PHILIPS_HUE_LAN_BRIDGE_IP=
PHILIPS_HUE_LIGHT_COLOR=
PHILIPS_HUE_LIGHT_IDS=
PHILIPS_HUE_LIGHT_PATTERN=
PHONE_CARRIER=tmobile
PHONE_NUMBER=
PLAY_SOUND=
PROXY_ADDRESS=
PROXY_PROTOCOL=
PROXY_PORT=
PUSHBULLET=
PUSHOVER_EXPIRE=
PUSHOVER_RETRY=
PUSHOVER_TOKEN=
PUSHOVER_USER=
PUSHOVER_PRIORITY=
SCREENSHOT=true
SHOW_ONLY_BRANDS=
SHOW_ONLY_MODELS=
SHOW_ONLY_SERIES=
SLACK_CHANNEL=
SLACK_TOKEN=
SMTP_ADDRESS=
SMTP_PORT=
STORES="amazon,amd,adorama,antonline,bestbuy,asus,bandh,corsair,evga,gamestop,newegg,nvidia,officedepot"
TELEGRAM_ACCESS_TOKEN=
TELEGRAM_CHAT_ID=
TWILIO_ACCOUNT_SID=
TWILIO_AUTH_TOKEN=
TWILIO_FROM_NUMBER=
TWILIO_TO_NUMBER=
TWITCH_ACCESS_TOKEN=
TWITCH_CHANNEL=
TWITCH_CLIENT_ID=
TWITCH_CLIENT_SECRET=
TWITCH_REFRESH_TOKEN=
TWITTER_ACCESS_TOKEN_KEY=
TWITTER_ACCESS_TOKEN_SECRET=
TWITTER_CONSUMER_KEY=
TWITTER_CONSUMER_SECRET=
TWITTER_TWEET_TAGS=
WEB_PORT=

**Comment by DigiBugCat** [comment]
Interesting, unfortunately I'm not able to replicate this issue on my machine (I don't have access to a windows development machine though). The above configuration does work as expected on ubuntu server 20.04 and macos 10.15.6. 

Hopefully someone else can figure out the issue, I have an inkling that it might be a path issue but I don't have enough knowledge about this project to give more information

**Comment by MJB1113** [comment]
@jef

**Comment by jef** [comment]
How about changing the dotenv just to STORES=amazon to test. Then go from there.

**Comment by MJB1113** [comment]
> How about changing the dotenv just to STORES=amazon to test. Then go from there.

hmm its quite strange, regardless of what I set the stores to it still shows that it searches amazon and bestbuy.

**Comment by MJB1113** [comment]
@jef do you have any idea of how I can fix this?

**Comment by theshaal** [comment]
I had the same issue on Windows as well. This seems to have something to do with how Windows saves the file. When it's saved as "All Files" via Notepad or some other text editors, the file is still saved as a Text Document for whatever reason, and that Text Document isn't read by the program properly. The fix I found was to copy the file exactly and never use "Save as", only using the regular "Save" to make the changes you need

**Comment by MJB1113** [comment]
> I had the same issue on Windows as well. This seems to have something to do with how Windows saves the file. When it's saved as "All Files" via Notepad or some other text editors, the file is still saved as a Text Document for whatever reason, and that Text Document isn't read by the program properly. The fix I found was to copy the file exactly and never use "Save as", only using the regular "Save" to make the changes you need

Im not exactly sure what you mean here. Can you elaborate?

**Comment by theshaal** [comment]
Check the dotenv file in your File Explorer. Does the type of the file say "File" or "Text Document"? If it says "Text Document", that seems to break the program, it has to stay of type "File"

**Comment by MJB1113** [comment]
> Check the dotenv file in your File Explorer. Does the type of the file say "File" or "Text Document"? If it says "Text Document", that seems to break the program, it has to stay of type "File"

my dotenv file says "env file" on the right side

**Comment by MJB1113** [comment]
![image](https://user-images.githubusercontent.com/45674525/103836251-8a24a900-5056-11eb-9d34-021e6201cd4c.png)
I do get these errors though

**Comment by jef** [comment]
Those warnings are expected. Everything looks fine there.

> hmm its quite strange, regardless of what I set the stores to it still shows that it searches amazon and bestbuy.

This tells me your environment file is incorrect. Can you take a screenshot of your explorer (where streetmerchant and the dotenv file is at)?

**Comment by sshokooh** [comment]
> > Check the dotenv file in your File Explorer. Does the type of the file say "File" or "Text Document"? If it says "Text Document", that seems to break the program, it has to stay of type "File"
> 
> my dotenv file says "env file" on the right side

Remove the .env file extension so that it of the generic type "file" and not of type "env".

**Comment by Carter-Thomas** [comment]
I have the same issue, I've done all the steps above, and still doesn't work, my dotenv-example is a File, not text. I did a regular "save" and not save as, and still nothing. I removed bestbuy as a test, and it still showed it in the startup.

**Comment by Carter-Thomas** [comment]
![2021-01-09](https://user-images.githubusercontent.com/77213551/104110407-f7298000-529c-11eb-97dc-354f15b3ff10.png)

The other streetmerchant file doesn't work just so you know
The weird thing is, a 6800 for 1300$ or something was on amazon, the auto add to cart changed with it, but the price limit nor the stores changed. 
Another edit: I also removed the .txt dotenv that appeared for some reason, but otherwise, I don't know why this is happening
Edit: FINALLY, I solved it @MJB1113 . So, you go to the c:\ version of the streetmerchant and switch out the example for the one you just made, follow everything you did before, and that should fix it.

**Comment by msakallah** [comment]
> ## Expected Behavior
> There should be 13 stores that are being searched
> 
> ## Current Behavior
> I have more than two stores selected however only those are showing up (in the picture below)
> 
> ## Steps to Reproduce
> ![image](https://user-images.githubusercontent.com/45674525/103664390-3c1d8180-4f40-11eb-8ae6-131a19a2f866.png)
> 
> ## Environment
> **OS:**
> **dotenv file:**
> 
> ```
> 
> ```
> 
> ## Logs

I just updated my local copy of the code and I am exhibiting the same issue. I thought it was some file that got corrupted as it was working until this morning. I tried to troubleshoot but didn't get anywhere.  

I deleted the local copy and cloned it again, and configured the dotenv, but still had the same issue.  No matter what I do it is no longer working. 

![image](https://user-images.githubusercontent.com/1391817/105113081-40928000-5a92-11eb-981d-1907d086f56d.png)


Here is my dotenv file (removed sensitive data)


ASCII_BANNER=
ASCII_COLOR=
AUTO_ADD_TO_CART=true
BROWSER_TRUSTED=
COUNTRY=
DESKTOP_NOTIFICATIONS=true
DISCORD_NOTIFY_GROUP=
DISCORD_NOTIFY_GROUP_3060TI=
DISCORD_NOTIFY_GROUP_3070=
DISCORD_NOTIFY_GROUP_3080=
DISCORD_NOTIFY_GROUP_3090=
DISCORD_NOTIFY_GROUP_CORSAIR_SF=
DISCORD_NOTIFY_GROUP_RX6800=
DISCORD_NOTIFY_GROUP_RX6800XT=
DISCORD_NOTIFY_GROUP_RX6900XT=
DISCORD_NOTIFY_GROUP_RYZEN5600=
DISCORD_NOTIFY_GROUP_RYZEN5800=
DISCORD_NOTIFY_GROUP_RYZEN5900=
DISCORD_NOTIFY_GROUP_RYZEN5950=
DISCORD_NOTIFY_GROUP_SONYPS5C=
DISCORD_NOTIFY_GROUP_SONYPS5DE=
DISCORD_NOTIFY_GROUP_XBOXSX=
DISCORD_NOTIFY_GROUP_XBOXSS=
DISCORD_NOTIFY_GROUP_TEST=
DISCORD_WEB_HOOK=
EMAIL_PASSWORD="(removed)"
EMAIL_TO="(removed)@gmail.com"
EMAIL_USERNAME="(removed)@gmail.com"
HEADLESS=
IN_STOCK_WAIT_TIME=
LOG_LEVEL=
LOW_BANDWIDTH=
MAX_PRICE_SERIES_3060TI=
MAX_PRICE_SERIES_3070=
MAX_PRICE_SERIES_3080="935"
MAX_PRICE_SERIES_3090=
MAX_PRICE_SERIES_CORSAIR_SF=
MAX_PRICE_SERIES_RX6800=
MAX_PRICE_SERIES_RX6800XT="850"
MAX_PRICE_SERIES_RX6900XT=
MAX_PRICE_SERIES_RYZEN5600=
MAX_PRICE_SERIES_RYZEN5800=
MAX_PRICE_SERIES_RYZEN5900=
MAX_PRICE_SERIES_RYZEN5950=
MAX_PRICE_SERIES_SONYPS5C=
MAX_PRICE_SERIES_SONYPS5DE=
MAX_PRICE_SERIES_XBOXSS=
MAX_PRICE_SERIES_XBOXSX=
MAX_PRICE_SERIES_TEST=
MICROCENTER_LOCATION="mayfield-heights"
MQTT_BROKER_ADDRESS=
MQTT_BROKER_PORT=
MQTT_CLIENT_ID=
MQTT_PASSWORD=
MQTT_QOS=
MQTT_TOPIC=
MQTT_USERNAME=
NVIDIA_ADD_TO_CART_ATTEMPTS=
NVIDIA_SESSION_TTL=
OPEN_BROWSER=
PAGE_BACKOFF_MIN=
PAGE_BACKOFF_MAX=
PAGE_SLEEP_MIN="2000"
PAGE_SLEEP_MAX="30000"
PAGE_TIMEOUT="0"
PAGERDUTY_INTEGRATION_KEY=
PAGERDUTY_SEVERITY=
PHILIPS_HUE_API_KEY=
PHILIPS_HUE_CLOUD_ACCESS_TOKEN=
PHILIPS_HUE_CLOUD_CLIENT_ID=
PHILIPS_HUE_CLOUD_CLIENT_SECRET=
PHILIPS_HUE_CLOUD_REFRESH_TOKEN=
PHILIPS_HUE_LAN_BRIDGE_IP=
PHILIPS_HUE_LIGHT_COLOR=
PHILIPS_HUE_LIGHT_IDS=
PHILIPS_HUE_LIGHT_PATTERN=
PHONE_CARRIER="verizon"
PHONE_NUMBER="removed"
PLAY_SOUND="game.mp3"
PROXY_ADDRESS=
PROXY_PROTOCOL=
PROXY_PORT=
PUSHBULLET=
PUSHOVER_EXPIRE=
PUSHOVER_RETRY=
PUSHOVER_TOKEN=
PUSHOVER_USER=
PUSHOVER_PRIORITY=
SCREENSHOT=
SHOW_ONLY_BRANDS="sapphire,gigabyte,asus"
SHOW_ONLY_MODELS="nitro+,gaming oc,strix oc"
SHOW_ONLY_SERIES="rx6800xt,3080"
SLACK_CHANNEL=
SLACK_TOKEN=
SMTP_ADDRESS=smtp.gmail.com
SMTP_PORT=465
STORES="asus,amazon,bandh,bestbuy,microcenter,newegg,target,walmart"
TELEGRAM_ACCESS_TOKEN=(removed)
TELEGRAM_CHAT_ID=(removed)
TWILIO_ACCOUNT_SID=
TWILIO_AUTH_TOKEN=
TWILIO_FROM_NUMBER=
TWILIO_TO_NUMBER=
TWITCH_ACCESS_TOKEN=
TWITCH_CHANNEL=
TWITCH_CLIENT_ID=
TWITCH_CLIENT_SECRET=
TWITCH_REFRESH_TOKEN=
TWITTER_ACCESS_TOKEN_KEY=
TWITTER_ACCESS_TOKEN_SECRET=
TWITTER_CONSUMER_KEY=
TWITTER_CONSUMER_SECRET=
TWITTER_TWEET_TAGS=
WEB_PORT=

**Reply by msakallah** [reply]
OK Here is an update. I deleted my entire copy of the code on my machine. I cloned the project again on my machine (new directory). I duplicated the dotenv-example and named the file dotenv. Then I modified it with only STORES="amazon" and SHOW_ONLY_SERIES="rx6800xt".  I still see the issue.

**Comment by wofnull** [comment]
please be so kind and remove the " from the STORES LINE, 

so that there is following visible:

STORES=asus,amazon,bandh,bestbuy,microcenter,newegg,target,walmart

same for brands / models / series

**Reply by msakallah** [reply]
I thought that we should have quotations around those.
Anyway, I did that as well and it didn't work. My dotenv had only the store value and series changed.

**Reply by wofnull** [reply]
please ensure that it looks like this, on my end with  the current code from main repo it works without issues:
![image](https://user-images.githubusercontent.com/449003/105607614-fd8d2100-5d9f-11eb-99e8-7870e4c5d9d6.png)

**Comment by monkey-d-uchiha1** [comment]
Make sure u have git installed and njs node or whatever it's called download windows visual and edit file on there and run program on there that's how I fixed mine windows command promt dosnt work well with this checker

**Comment by lec6213** [comment]
I hope this isn't too obvious but if you install via CMD, windows command, it will most likely install into C:/StreetMerchant.  If you install it via GIT BASH, while in the GIT console it will install into /Users/YOUSELF/Streetmerchant/

Could this be you installed it via Windows or GIT and are looking at another install and running a different DOTENV file based on weather you are launching via GIT or Windows CMD?

I would check both paths above, if you have streetmerchant in both you installed both a GIT and Windows version.  Tell us are you "NPM RUN START" in CMD or in GIT, that will tell us what directory the DOTENV file is that is being used...

I would wager you have a second install and are editing the wrong DOTENV

**Comment by msakallah** [comment]
I apologize for the delay in my response to this issue. I was out of town. 
I just want to emphasize a few things before I proceed with my answer:
- I have only one installation of this code
- This code worked fine until I updated it using instructions found @ [https://github.com/jef/streetmerchant/wiki/Installation:-Windows#bot-update](url) on Jan 19th. I update the local code once a week
- If I restore a backup of the old code prior to the update, everything works as expected
- each attempt to fix this issue I have deleted the old code and reinstalled from scratch.

But here are my responses
@lec6213 : I have attempted to install this code from source using instructions found @ [https://github.com/jef/streetmerchant/wiki/Installation:-Windows](url).  I did that using CMD, as well as PowerShell.   I have also attempted to install using GitHub Desktop
Also, to be on the safe side I validated that I do not have anything under my profile in C:\USERS. Additionally, my code is installed on D:\StreetMerchant

@wofnull & @monkey-d-uchiha1 I have attempted your suggestions. I put in just Amazon as my store (to test) without quotation, and put in the hardware to scan for also without quotation. The results are still the same. 
The file type is FILE. I have attached a screenshot showing all 3 (DOTENV, Explorer View, run results)
![2021-01-26 02_36_00-](https://user-images.githubusercontent.com/1391817/105818647-2a4e5d80-5f85-11eb-9d4a-83216e218227.png).

What has changed code-wise that may cause this? as indicated It was running fine until I updated on the 19th around 3am. Prior update was on the 17th and had no issues

**Reply by jef** [reply]
Try and remove _everything_ in `dotenv` except `STORES=amazon`. Start by using `npm run start` instead of `npm run start:dev`.

**Reply by msakallah** [reply]
Thanks Jef, again I apologize for not responding sooner.
However, I finally got back to it. I started the script as you instructed using only `npm run start` and that did the trick, without modifying my dotenv file. So how come `npm run start:dev` is causing issues now?
